ocs_ci/ocs/platform_nodes.py

```
@@ -3874,3 +3878,331 @@ def create_nodes(self, node_conf, node_type, num_nodes):
+                # Construct full node name
+                node_name = f"{node_role}.{rack_serial}.fusion.tadn.ibm.com"
+                try:
+                    logger.info(f"Powering on node: {node_name}")
+                    # Use power_on_direct which doesn't require API access
+                    result = self.ibm_hci.power_on_direct(node_name)
+                    if result:
+                        powered_on_nodes.append(node_name)
+                        logger.info(f"Successfully powered on: {node_name}")
+                    else:
+                        logger.warning(f"Failed to power on {node_name}")
+                except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)